pkg/cnab/cnab-to-oci/registry.go

```diff
@@ -43,7 +43,7 @@ func (r *Registry) PullBundle(tag string, insecureRegistry bool) (*bundle.Bundle
 		insecureRegistries = append(insecureRegistries, reg)
 	}
 
-	bun, err := remotes.Pull(context.Background(), ref, r.createResolver(insecureRegistries))
+	bun, _, err := remotes.Pull(context.Background(), ref, r.createResolver(insecureRegistries))
```

Comment:
A relocation map is generated by `Pull` - right now we are not using it (so the behaviour of `porter pull` is identical).